tests/unit/gpu_telemetry/test_metrics_config.py

```
@@ -309,22 +314,24 @@ def test_metrics_removed_from_defaults_are_added_from_csv(self):
+            assert (
+                DCGM_TO_FIELD_MAPPING["DCGM_FI_DEV_MEMORY_TEMP"] == "nvidia_memory_temp"
+            )
             assert (
                 DCGM_TO_FIELD_MAPPING["DCGM_FI_DEV_POWER_MGMT_LIMIT"]
-                == "power_mgmt_limit"
+                == "nvidia_power_mgmt_limit"
             )
 
             # Verify the auto-generated field names are correct
             for metric in custom_metrics:
                 display_name, field_name, unit = metric
```

Ruff (0.15.15)

[warning] Unpacked variable `display_name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)